types/jasmine/index.d.ts

```diff
@@ -1182,30 +1252,58 @@ declare module "jasmine" {
         onCompleteCallbackAdded: boolean;
         defaultReporterConfigured: boolean;
 
-        constructor(options: any);
+        constructor(options: jasmine.JasmineOptions);
         addMatchers(matchers: jasmine.CustomMatcherFactories): void;
+        /**
+         * Add a custom reporter to the Jasmine environment.
+         */
         addReporter(reporter: jasmine.CustomReporter): void;
+        /**
+         * Adds a spec file to the list that will be loaded when the suite is executed.
+         */
         addSpecFile(filePath: string): void;
         addSpecFiles(files: string[]): void;
         addHelperFiles(files: string[]): void;
         addRequires(files: string[]): void;
-        configureDefaultReporter(options: any, ...args: any[]): void;
+        /**
+         * Configure the default reporter.
+         */
+        configureDefaultReporter(options: jasmine.DefaultReporterOptions): void;
         execute(files?: string[], filterString?: string): Promise<void>;
         exitCodeCompletion(passed: boolean): void;
-        loadConfig(config: any): void;
+        loadConfig(config: jasmine.JasmineConfig): void;
         loadConfigFile(configFilePath?: string): void;
         loadHelpers(): Promise<void>;
         loadSpecs(): Promise<void>;
         loadRequires(): void;
         onComplete(onCompleteCallback: (passed: boolean) => void): void;
+        /**
+         * Provide a fallback reporter if no other reporters have been specified.
+         */
         provideFallbackReporter(reporter: jasmine.CustomReporter): void;
+        /**
+         * Clears all registered reporters.
+         */
         clearReporters(): void;
-        randomizeTests(value?: boolean): void;
+        /**
+         * Sets whether to randomize the order of specs.
```

Comment:
```suggestion
         * Sets whether to randomize the order of specs.
         * @deprecated	    
```
https://github.com/jasmine/jasmine/blob/163f93d6ffe905a7d2f907f83cea934706b69e64/src/core/Env.js#L541

Reply:
This function isn't deprecated https://github.com/jasmine/jasmine-npm/blob/main/lib/jasmine.js#L70-L78 only `jasmine.env.randomizeTests()` is

Reply:
sry, same topic (I've looked into wrong place, should catch up with optitian r somethng)